core/src/main/java/com/scalar/db/storage/jdbc/JdbcOperationAttributes.java

```diff
@@ -0,0 +1,49 @@
+package com.scalar.db.storage.jdbc;
+
+import com.scalar.db.api.Delete;
+import com.scalar.db.api.Put;
+import java.util.Map;
+
+/** A utility class to manipulate the operation attributes for JDBC. */
+public final class JdbcOperationAttributes {
+  private static final String OPERATION_ATTRIBUTE_PREFIX = "jdbc-";
+
+  public static final String
+      LEFT_OUTER_VIRTUAL_TABLE_PUT_IF_IS_NULL_ON_RIGHT_COLUMNS_CONVERSION_ENABLED =
+          OPERATION_ATTRIBUTE_PREFIX
+              + "left-outer-virtual-table-put-if-is-null-on-right-columns-conversion-enabled";
+
+  public static final String LEFT_OUTER_VIRTUAL_TABLE_DELETE_IF_IS_NULL_ON_RIGHT_COLUMNS_ALLOWED =
+      OPERATION_ATTRIBUTE_PREFIX
+          + "left-outer-virtual-table-delete-if-is-null-on-right-columns-allowed";
```

Comment:
The operation attribute names are indeed quite long, but I wasn’t able to come up with reasonable alternatives.